models/custom/snowplow_media_player_session_stats.sql

```diff
@@ -33,6 +33,7 @@ with prep as (
       cast(({{ snowplow_utils.get_split_to_array('domain_sessionid_array', 'b') }})[1] as {{ type_string() }}) as domain_sessionid,
     {% endif %}
     user_identifier,
+    app_id, 
```

Comment:
Tracker engineers have confirmed that there may be cases where there are different app_ids that may belong to the same session. We should take the first or last here (or not add it here as it is not needed by the app).

Reply:
I think it is better to remove over picking the last.